ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -38,6 +40,9 @@ public class BottomSheetController: UIViewController {
     /// Sheet content below the header which is only visible when the sheet is expanded.
     @objc public let expandedContentView: UIView
 
+    /// Indicates if the main content is dimmed when the sheet is expanded.
+    @objc public let shouldShowDimmingView: Bool
```

Comment:
its not like clients can change it after they initialize it. is there a reason we are making this public?